forecr-dsb-ornx-orin-nano-4gb.coffee

```diff
@@ -0,0 +1,43 @@
+deviceTypesCommon = require '@resin.io/device-types/common'
+{ networkOptions, commonImg, instructions } = deviceTypesCommon
+
+BOARD_PREPARE  = 'Put the board in recovery mode'
+FLASH_TOOL = 'Unzip BalenaOS image and use <a href=\"https://github.com/balena-os/jetson-flash\">Jetson Flash</a> to provision the device.'
+DONE_FLASHING  = 'After flashing is completed, please wait until the board is rebooted'
+module.exports =
+	version: 1
+	slug: 'forecr-dsb-ornx-orin-nano-4gb'
+	name: 'Forecr DSB ORNX / Orin Nano 4GB'
```

Comment:
Hi @asuykov , the module in this DT will be an Orin Nano 4GB?

Reply:
In this particular config, yet. And it's also the one I've got plugged in there right now.

It's a carrier board which takes any Orin Nano or NX SoM.

Reply:
thanks @asuykov . What module will the final device-type use? A Nano 4GB, 8GB or NX 16GB?

Reply:
Two modules, Nano 8GB and NX 16GB. This was explicitly requested, the project will need both. So it will need two device-types at least I guess. I also did Nano 4GB since I have the module.

I've got all three modules at hand, but only one carrier board, so any testing or anything else really will be one module at a time.